3-nodes-mirror-3-dc/files/config.yaml

```diff
@@ -129,10 +132,21 @@ interconnect_config:
     path_to_private_key_file: "/opt/ydb/certs/node.key"
     path_to_ca_file: "/opt/ydb/certs/ca.crt"
 grpc_config:
+    ssl_port: 2135
```

Comment:
IIRC, it's currently set via `ydbd` CLI arguments because there are multiple nodes per server, and they use different ports but the same config.

Reply:
port definition is removed